reco_utils/dataset/wikidata.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import requests
+
+def find_wikidataID(name):
+    """Find the entity ID in wikidata from a title string.
+    Args:
+        name (str): A string with a wikidata page title.
+    Returns:
+        entityID: wikidata entityID corresponding to the title string. 
+                  'entityNotFound' will be returned if no page is found
+    """
+    url = "https://en.wikipedia.org/w/api.php?action=query&prop=pageprops&format=json&titles="
+    r = requests.get(url+name.replace(" ", "%20"))
+    entityID = list(r.json()["query"]["pages"].keys())[0]
```

Comment:
it's possible here too that the result may not be formed as expected, particularly if it is an error, this will also generate a TypeError or KeyError exception which calling functions will need to handle.

Reply:
In this commit I changed the exceptions to catch requests exceptions and used .get for the dictionaries: 51b0fd2